tests/granulate_utils/metrics/fixtures.py

```diff
@@ -85,6 +86,9 @@ def _mock_subprocess_run_stdout(self: T, *args: Any, **kwargs: Any) -> Mock:
         self._mock.stdout = self._stdout[cmd]
         return self._mock
 
+    def mock_path_is_dir(self) -> None:
+        self._contexts.add(patch("pathlib.Path.is_dir", lambda path: True))
```

Comment:
Mocking all dirs doesn't make sense to me. This can create problems or hide problems. Mock only the particular things you need